### DISCUSSION TITLE
What does this host parameter do?

### DISCUSSION THREAD

**Discussion by ByronAP** [Chia-Network/chia-blockchain#8679] [Q&A]
What does this host parameter do? I can't seem to  track exactly where it is being used and or for what.

Anyone know? it seems any value is acceptable.

https://github.com/Chia-Network/chia-blockchain/blob/66c2399ad5cedafed8fa8af7b61d1eda8082859a/chia/rpc/wallet_rpc_client.py#L27

**Comment by loppefaaret** [comment]
it is an online backup of wallet metadata, used for advanced wallet setups, like future DID wallets, rate limited wallet and so on, the basic wallet does not make use of this function

**Comment by BrandtH22** [comment]
@ByronAP , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
